carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/device/Device.java

```diff
@@ -712,9 +712,9 @@ public String getSysLog() {
          // adb -s UDID logcat -d
             @Override
             public String call() throws Exception {
-                LOGGER.info("Start Syslog extraction");
+                LOGGER.debug("Start Syslog extraction");
                 String[] cmd = CmdLine.insertCommandsAfter(executor.getDefaultCmd(), "-s", getAdbName(), "logcat", "-d");
-                LOGGER.info("Logcat log has been extracted.");
+                LOGGER.debug("Logcat log has been extracted.");
                 StringBuilder tempStr = new StringBuilder();
                 executor.execute(cmd).stream().forEach((k) -> tempStr.append(k.concat("\n")));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "k" parameter (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1611)